<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-S76SA9N" failures="1" tests="1" name="runner.HRISDrawSignatureRunner" time="17.197" errors="0" timestamp="2022-07-07T00:59:04 ICT" skipped="0">
  <testcase classname="runner.HRISDrawSignatureRunner" name="runScenario" time="17.197">
    <failure message="expected [Succesfully uploaded] but found [Alert is Not Displayed]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Succesfully uploaded] but found [Alert is Not Displayed]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at definitions.data.HRISBukcetDataDefine.validateSignature(HRISBukcetDataDefine.java:276)
at âœ½.User validate signture(file:///D:/Project/Java/TestNG/HRIS/src/test/resources/features/HRISDrawSignature.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.HRISDrawSignatureRunner -->
